testsuite/forge-cli/src/main.rs

```diff
@@ -2026,6 +2030,12 @@ fn realistic_env_max_load_test(
                 serde_yaml::to_value(OnChainExecutionConfig::default_for_genesis())
                     .expect("must serialize");
         }))
+        .with_validator_override_node_config_fn(Arc::new(move |config, _| {
+            optimize_state_sync_for_throughput(config, 2000);
```

Comment:
what is the default? should we increase the default ?

We are increasing block size on mainnet to 3000 with 1.18

Reply:
The default is 1k. Upgrading mainnet isn't as simple -- we'll need to bump the storage side limits first, before the clients. I'll also need to double check that we aren't in execution fallback territory regarding output sizes. I'll send out a PR to bump the server side first.

FWIW, I'm not super worried about it because consensus observer is going to be enabled in 1.18 😄